src/cljs/xt_play/model/interval.cljs

```diff
@@ -0,0 +1,30 @@
+(ns xt-play.model.interval
+  (:require [re-frame.core :as rf]))
+
+(defonce interval-handler
+  (let [live-intervals (atom {})]
+    (fn handler [{:keys [action id freq event]}]
+      (case action
+        :clean   (doall (map #(handler {:action :end  :id  %1}) (keys @live-intervals)))
+        :start   (swap! live-intervals assoc id (js/setInterval #(rf/dispatch event) freq))
+        :end     (do (js/clearInterval (get @live-intervals id)) 
+                     (swap! live-intervals dissoc id))))))
+
+;; when this code is reloaded `:clean` existing intervals
+(interval-handler {:action :clean})
+
+(rf/reg-fx ::interval interval-handler)
```

Comment:
I'd probably have three `reg-fx`'s: `::start`, `::stop` & `::clean` if I were imagining this as an external library.
This is completely fine though, especially as it's only used in this namespace

Reply:
I've added a `::clean`, `::start` and `::stop` should, imo, be explicit to the id